core/src/test/scala/tech/beshu/ror/unit/acl/factory/decoders/rules/elasticsearch/FieldsRuleSettingsTest.scala

```diff
@@ -288,7 +288,7 @@ class FieldsRuleSettingsTest extends BaseRuleSettingsDecoderTest[FieldsRule] {
               |""".stripMargin,
           assertion = errors => {
             errors should have size 1
-            errors.head should be(GeneralReadonlyrestSettingsError(Message("Unknown fls engine: 'something'. Supported: 'es_with_lucene', 'es'.")))
+            errors.head should be(GeneralReadonlyrestSettingsError(Message("Unknown fls engine: 'something'. Supported: 'lucene', 'es', 'es_with_lucene'(default).")))
```

Comment:
AFAIK we don't officially say that `lucene` mode is supported. We left it as a "last resort" option, but it should never be needed to be used. Let's add proper comment somewhere and let's revert the previous message